test/stress_tests/test_actor_event_loop.py

```diff
@@ -0,0 +1,119 @@
+import ray
+import numpy as np
+import time
+from collections import Counter
+import json
+import sys
+
+LOCAL = len(sys.argv) > 1
```

Comment:
not a huge deal, but we use `argparse` everywhere else.

Maybe put the actual application code inside of `if __name__ == "__main__":`